stdlib/database/sql/src/main/ballerina/src/sql/Module.md

```diff
@@ -340,5 +340,30 @@ if (ret is error) {
 }
 ```
 
+#### Execute SQL stored procedures
+
+This example demonstrates how to execute a stored procedure with a single INSERT statement that is executed via the 
+`call` remote function of the client.
+
+```ballerina
+int uid = 10;
+sql:OutParameter insertId = new;
+
+var ret = dbClient->call(`call InsertPerson(${uid}, ${insertId}`);
+if (ret is error) {
+    io:println("Error occurred:", err.message());
+} else {
+    io:println("Outparameter insert id: ", insertId.get(int));
+    stream<record{}, sql:Error>? resultStr = result.queryResult;
```

Comment:
Is this working sample? What is `result` variable here?

Reply:
Seems to be an error in the sample, it should be `ret` variable returned from the call api. Fixed in c8978fd